MachineLearning_Engine/Compute/LinearRegression.cs

```diff
@@ -57,5 +60,21 @@ public static Tensor Infer(LinearRegression model, Tensor x)
         }
 
         /*************************************/
+
+        [Description("Projects the given input using a linear regression model")]
+        [Input("model", "The linear regressor model used for inference")]
+        [Output("")]
+        public static Output<List<double>, double> Coefs(LinearRegression model)
```

Comment:
Maybe we can move the method to the `Query` class?
If we name it `Weights` or `Coefficients` in a file like `Coefficients.cs` this can also be more scalable in the future to include SVCs, SVRs, DNNs, KMeans and so on :)